portfolio/src/main/webapp/hello.html

```diff
@@ -0,0 +1,6 @@
+So I'm hoping that by creating a new .html file called "hello",
```

Comment:
Bare text nodes are not well-structured HTML. Also, this file is not well-structured HTML: e.g. starts with a `<html>` etc.

Reply:
You have added some structure with this commit but you still have a bare text node.